CHANGELOG.md

```diff
@@ -50,6 +50,9 @@ This project adheres to [Semantic Versioning](https://semver.org/).
    * Deprecated `pysat.Instrument._filter_netcdf4_metadata` and replaced it
      with `pysat.utils.io.filter_netcdf4_metadata`.
    * Changed `fname` from a kwarg to an arg in `pysat.Instruments.to_netcdf4`
+   * Deprecated support for 2D pandas datasets
```

Comment:
```suggestion
   * Deprecated support for 2D pandas datasets in `pysat._instrument`
```
How about this?

Reply:
I mean, it should be deprecated everywhere in this PR, not just in one place.